GOAT-Compiler/RecSymbolTable.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GOAT_Compiler
+{
+    class RecSymbolTable : ISymbolTable
+    {
+
+        private Table tables;
+        private Table mainTable;
```

Comment:
maybe call it globaltable cause its the global scope

Reply:
Yes Sir!
Renamed to globalScope